packages/react-magma-dom/src/components/Pagination/SimplePagination.test.js

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import { axe } from '../../../axe-helper';
+import { magma } from '../../theme/magma';
+import { Pagination } from '.';
+import { render, fireEvent } from '@testing-library/react';
+import { transparentize } from 'polished';
+
+const testId = 'test-id';
+
+describe('Simple Pagination', () => {
+  it('Should find element by testId', () => {
+    const { getByTestId } = render(<Pagination simple testId={testId} />);
+
+    expect(getByTestId(testId)).toBeInTheDocument();
+  });
+
+  it('Should render a native select', () => {
+    const { getByText } = render(<Pagination count={4} simple />);
+
+    const selected = getByText('2').parentElement.parentElement;
+
+    expect(selected).toHaveStyleRule(
+      'background-color',
+      magma.colors.neutral100
+    );
+    expect(selected).toHaveStyleRule('border-radius', '8px');
+  });
+
+  it('Should render an inverse select', () => {
+    const { getByText } = render(<Pagination count={4} simple isInverse />);
+
+    const selected = getByText('2').parentElement.parentElement;
+
+    expect(selected).toHaveStyleRule(
+      'background-color',
+      transparentize(0.8, magma.colors.neutral900)
+    );
+  });
+
+  it('Should render disabled navigation icons, text, and select', () => {
+    const { getByText, getByLabelText } = render(
+      <Pagination simple count={4} disabled />
+    );
+    const previousButton = getByLabelText('Previous Page');
+    const nextButton = getByLabelText('Next Page');
+
+    const selected = getByText('2').parentElement;
+
+    expect(selected).toHaveStyleRule('cursor', 'not-allowed');
+
+    expect(selected).toHaveStyleRule(
+      'color',
+      transparentize(0.4, magma.colors.neutral500)
+    );
+
+    expect(previousButton).toHaveStyleRule(
+      'color',
+      transparentize(0.4, magma.colors.neutral500)
+    );
+    expect(nextButton).toHaveStyleRule(
+      'color',
+      transparentize(0.4, magma.colors.neutral500)
+    );
+  });
+
+  it('Should render inverse disabled navigation icons, text, and select', () => {
+    const { getByText, getByLabelText } = render(
+      <Pagination simple count={4} isInverse disabled />
+    );
+    const previousButton = getByLabelText('Previous Page');
+    const nextButton = getByLabelText('Next Page');
+
+    const selected = getByText('2').parentElement;
+
+    expect(selected).toHaveStyleRule('cursor', 'not-allowed');
+
+    expect(selected).toHaveStyleRule(
+      'color',
+      transparentize(0.6, magma.colors.neutral100)
+    );
+
+    expect(previousButton).toHaveStyleRule(
+      'color',
+      transparentize(0.7, magma.colors.neutral100)
+    );
+    expect(nextButton).toHaveStyleRule(
+      'color',
+      transparentize(0.7, magma.colors.neutral100)
+    );
+  });
```

Comment:
Just a style recommendation, but I like using `describe`s to group together similar tests, or tests related to the same prop. So these could be wrapped in
```
describe('disabled', () => { 
   it ('', () => {});
   it ('', () => {});
})
```
I think it helps organize them and it's easier to see props have what coverage. I did this in ButtonGroup tests, for example.